webpack.config.js

```diff
@@ -28,6 +31,9 @@ module.exports = {
     },
     plugins: [
         new webpack.IgnorePlugin(/^electron$/),
-        new webpack.BannerPlugin({ banner: "#!/usr/bin/env node", raw: true})
+        new webpack.BannerPlugin({ banner: "#!/usr/bin/env node", raw: true}),
+        new CopyWebpackPlugin([
```

Comment:
If I understand this correctly we're using the plugin to copy a file. Could we do this in bash since we already do most of the packaging work there?

Reply:
I can, but when you say bash do you mean the `package.json`? If so, I can just add ` && cp bin/conjure-typescript.bat dist/bin` to `build`

Reply:
Done